src/main/scala/com/snowplowanalytics/indicative/LambdaHandler.scala

```diff
@@ -52,25 +56,60 @@ class LambdaHandler {
             EventTransformer
               .transformWithInventory(dataArray)
               .leftMap(errors => TransformationError(errors.mkString("\n  * "))))
-          indicativeEvent <- EitherT(Transformer.transform(snowplowEvent.event, snowplowEvent.inventory))
+          indicativeEvent <- EitherT(Transformer
```

Comment:
FYI, there `toEitherT` extension method on `Either`, which I believe makes it more readable.

Reply:
here it's an option so it doesnt work